phpunit.xml.dist

```diff
@@ -6,10 +6,6 @@
     backupStaticAttributes="false"
     colors="true"
 >
-    <php>
-        <env name="SYMFONY_DEPRECATIONS_HELPER" value="max[self]=999999"/>
-    </php>
-
```

Comment:
Can you quickly describe that change?

Reply:
the codebase no longer triggers deprecation warnings so we no longer need that dependency?